frontend/mobilityai_react/App/Screens/NewPatientForm.js

```diff
@@ -153,41 +163,63 @@ export default class PatientForm extends React.Component {
             <View style={styles.container}>
                 <ScrollView contentContainerStyle={styles.scrollable}>
                     <View style={{ paddingBottom: 100 }}>
+                        <Text style={styles.sliderText}>First Name:</Text>
                         <Field onChangeText={(firstName) => this.setState({ firstName })}
                             value={this.state.firstName}
                             placeholder="Enter First Name..."
                         />
+                        <Text style={styles.sliderText}>Last Name:</Text>
                         <Field onChangeText={(lastName) => this.setState({ lastName })}
                             placeholder="Enter Last Name..."
                             value={this.state.lastName}
                         />
-                        <Text style={styles.sliderText}>Daily time spent standing: {this.state.baselineStand} hours</Text>
-                        <SliderField onChange={(baselineStand) => this.setState({ baselineStand })}
-                            onSlidingComplete={(baselineStand) => this.setState({ baselineStand })}
-                            maximumValue={maxHours}
-                            step={baselineStepSize}
+                        <Text style={styles.sliderText}>Daily Hours spent standing:</Text>
+                        <Field 
+                            placeholder="Daily time spent standing..."
```

Comment:
This number is going to be changed to represent the target minutes spent standing per hour (0-60), so you should change the text and validation accordingly.

Reply:
This also applies for time sitting, time walking, etc.